pages/implementation/connect-inspector-to-segment.mdx

```diff
@@ -0,0 +1,227 @@
+---
+layout: 'guide'
+title: 'Send data from Segment to Avo Inspector (no code needed)'
+---
+
+import Link from '../../components/Link';
+
+# Enabling Avo Inspector without code changes
+
+## Use Avo Inspector as Segment destination
```

Comment:
Having these two titles in succession makes the top of the page a bit heavy. 

<img width="1280" alt="Screenshot 2021-05-27 at 15 09 58" src="https://user-images.githubusercontent.com/4923531/119851914-5ca1a480-befe-11eb-80fc-c686b9ba45c3.png">


Could we use the `abstract` field in the mdx header for either one of these titles?